benefits/working_hours.md

```diff
@@ -0,0 +1,13 @@
+# Working Hours
+
+We care about the quality of the work we produce rather than the amount of hours worked. We believe we individually know how we work best and can therefore set our own hours.
+
+Some things to note:
+
+ - We do not have a set amount of hours you need to work in a day or a week. Some people will work 6 hours, some 7, sometimes more than 8. Many of us will vary our hours day to day whilst others prefer routine.
+ - We do not have a specific start time but we do have our All Hands three times a week at 0950 so you should get into the office or dial in before that time.
+ - Likewise there isn't a time we expect everyone to leave the office. Of course if you're leaving at 1630 every day and are having problems delivering on your commitments someone will likely have a work with you to see if we can help improve things.
+ - Pressure can creep in around deadlines and sometimes staying late to ensure a smooth launch or to help diagnose and fix a problem is a necessity. However these occassions should be very rare. If you regularly feel this pressure say more than twice a year then speak to a colleague.
```

Comment:
Agree this pressure is usually caused by {unknowingly} dishonest estimation, which presents either inaccurate and/or imprecise estimates to customers.